tools/rum-models-generator/Tests/rum-models-generator-coreTests/RUMModelsGeneratorTests.swift

```diff
@@ -0,0 +1,32 @@
+/*
+* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+* This product includes software developed at Datadog (https://www.datadoghq.com/).
+* Copyright 2019-2020 Datadog, Inc.
+*/
+
+import XCTest
+@testable import RUMModelsGeneratorCore
+
+final class RUMModelsGeneratorTests: XCTestCase {
+    /// Test made for debugging purpose.
```

Comment:
In that code commented intentionally or maybe some leftover of local work?

Reply:
This is intentional. By enabling this test, we can make the tool run against real `rum-events-format` schemas - to test the code printed to console or to debug the tool if needed. I can imagine that if at some point `make rum-models-verify` or `make rum-models-generate` fails, we may want to look in there for more details, do the fix and run this test to see if all is good.